lib/pf/domain/ntlm_cache.pm

```diff
@@ -270,16 +273,23 @@ sub cache_user {
     my $config = $ConfigDomain{$domain};
     my $source = getAuthenticationSource($config->{ntlm_cache_source});
     return ($FALSE, "Invalid LDAP source $config->{ntlm_cache_source}") unless(defined($source));
-    
-    if($username =~ /^host\//) {
-        ($username, my $msg) = $source->findAtttributeFrom("servicePrincipalName", $username, "sAMAccountName");
-        return ($FALSE, $msg) unless($username);
+    my $cache_key = "$domain.$username";
```

Comment:
You're creating the cache key in many places with the same format.

Create a sub to encapsulate the cache key creation in case we change the format

Reply:
I am doing it twice but not really sure it's necessary to create a sub for that since i will do :
my $cache_key = create_cache_key($domain,$username);
and on the other place:
my $cache_key = create_cache_key($domain,$radius_username);

instead of:
my $cache_key = "$domain.$username";
and:
my $cache_key = "$domain.$radius_username";